.circleci/config.yml

```diff
@@ -0,0 +1,31 @@
+# Javascript Node CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2
+jobs:
+  build:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/node:10.12-stretch-browsers
+    steps:
+      - checkout
+      
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - node10-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - node10-dependencies-
+
+      - run: npm install
+      - run: ./node_modules/.bin/jspm install
+
+      - save_cache:
+          paths:
+            - node_modules
+          key: node10-dependencies-{{ checksum "package.json" }}
+
+      # run tests!
+      - run: node ./node_modules/gulp/bin/gulp.js build
+      - run: node ./node_modules/gulp/bin/gulp.js test
```

Comment:
Could reuse the [orb](https://github.com/aurelia/circleci/blob/master/src/v1.yml#L8) here if you just add package.json scripts that map to these gulp commands as well as prepare-release, so in package.json:

```json
"scripts": {
  "test": "gulp test",
  "build": "gulp build",
  "cut-release": "gulp prepare-release"
}
```

And then the entire config.yml can be reduced down to this:

```yml
version: 2.1

orbs:
  v1: aurelia/v1@volatile

workflows:
  main:
    jobs:
      - v1/build_test
```

We are trying to normalize all vCurrent repos like this (about 18 of the 40 or so repos are currently migrated to this new workflow). So if you're willing to just make this change to this particular repo then we'll be on 19 :)

If you ran out of time that's np, I'll just do it later then.

Reply:
Ok that sounds like a great improvement. I will give it a shot. I noticed the other PR for fixing the CI was merged so it might take me a bit more time to get things in order. I will close this PR and start a new one instead.